packages/frontend/package.json

```diff
@@ -26,6 +26,7 @@
     "ilp-packet": "3.1.4-alpha.2",
     "isbot": "^5.1.23",
     "json-canonicalize": "^1.0.6",
+    "next": "^15.2.3",
```

Comment:
```shell
rafiki|main ⇒ pnpm -r why next
Legend: production dependency, optional only, dev only

frontend /Users/max/interledger/rafiki/packages/frontend

dependencies:
@ory/integrations 1.3.1
└── next 15.2.2 peer
```

next is a dependency of @ory/integrations, so instead of adding it here as a direct dependency of the project, we can specify the override for next in `pnpm.overrides` in the root package json, such that we ensure a certain version (in this case, anything above 15.2.3) gets used when resolving dependencies.